dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/ContentMap.java

```diff
@@ -131,6 +133,8 @@ public Object getRaw(String fieldVariableName) {
 	
 	private Object get(String fieldVariableName, Boolean parseVelocity) {
 		try {
+			final boolean respectFrontEndRoles = UtilMethods.isSet(context.get("request")) ? PageMode.get(
```

Comment:
there is still a chance of an NPE on `context.get`.  This can be shorted to:
```
final boolean respectFrontEndRoles = Try.of(()->(HttpServletRequest)context.get("request")).getOrNull()).respectAnonPerms;
```